components/hackathon/common/HackathonWinners.tsx

```diff
@@ -0,0 +1,90 @@
+import { Flex, Grid, Icon, Image, Link, VStack } from "@chakra-ui/react";
+import { FiExternalLink } from "react-icons/fi";
+import { Heading, Text } from "tw-components";
+
+interface IWinner {
```

Comment:
```suggestion
interface Winner {
```

We don't use the I thing anywhere (or we try not to)

Reply:
my bad, old habit 🥲